text/chapter3.md

```diff
@@ -547,11 +579,23 @@ Note that, just like for top-level declarations, it was not necessary to specify
 
 ## Infix Function Application
 
-In the code for `findEntry` above, we used a different form of function application: the `head` function was applied to the expression `filter filterEntry book` by using the infix `$` symbol.
+Most of the functions discussed so far used _prefix_ function application, where the function name was put _before_ the arguments. For example, when using the `findEntry` function to search an `AddressBook`, one might write:
+
+```text
+> findEntry "John" "Smith" addressBook
+```
+
+However, this chapter has also included examples of _infix_ functions, such as  the `==` function in the definition of `filterEntry`, where the function is put _between_ the arguments. These infix operators are actually defined in the PureScript source as infix aliases for their underlying implementations. For example, `==` is defined as an alias for the prefix `eq` function with the line:
+
+```haskell
+infix 4 eq as ==
+```
+
+Likewise, in the code for `findEntry` above, we used a different form of function application: the `head` function was applied to the expression `filter filterEntry book` by using the infix `$` symbol.
 
 This is equivalent to the usual application `head (filter filterEntry book)`
 
-`($)` is just an alias for a regular function called `apply`, which is defined in the Prelude. It is defined as follows:
+`$` is just an alias for a regular function called `apply`, which is defined in the Prelude. It is defined as follows:
```

Comment:
```suggestion
`($)` is just an alias for a regular function called `apply`, which is defined in the Prelude. It is defined as follows:
```
Technically more correct as it was:
`($)` is an alias for `apply`
`$` is an alias for `` `apply` ``

Reply:
Ohh... I see now. Sorry about that.

Reply:
You could add a note clarifying those parens.
Although I'd rather just point readers to the language spec for the nitty-gritty details.